### PR TITLE
Add check for presence of required labels

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

File: roles/parse_operator_bundle/tasks/main.yml [hunk 1]
```diff
@@ -17,6 +17,18 @@
...
+        failed_label: "{{ item }}"
+      when: item is not in skopeo_inspect_json.Labels
+      with_items: "{{ required_labels }}"
+    - fail:
+        msg: "Label not found {{ failed_label }}"
+      when: failed_label != ""
+
   - name: "Set the main operator bundle info as parsed from the bundle image labels"
     set_fact:
       is_bundle_image: "{{ skopeo_inspect_json.Labels['com.redhat.delivery.operator.bundle'] }}"
```
Comment (lslebodn) [inline roles/parse_operator_bundle/tasks/main.yml:34]: Do we want to change also this line?

File: roles/parse_operator_bundle/tasks/main.yml [hunk 2]
```diff
@@ -17,6 +17,18 @@
     set_fact:
       skopeo_inspect_json: "{{ skopeo_inspect_result.stdout | from_json }}"
 
+  - name: "Verify required labels"
+    block:
+    - set_fact:
+        failed_label: ""
+    - set_fact:
+        failed_label: "{{ item }}"
```
Comment (dirgim) [inline roles/parse_operator_bundle/tasks/main.yml:25]: Suggestion: we may want to aggregate all missing labels here in case more than one is missing.

**Comment by dirgim** [comment]
Closing in favor of #35

**Pull request closed without merging**